src/otx/core/model/segmentation.py

```diff
@@ -217,6 +226,57 @@ def _dispatch_label_info(label_info: LabelInfoTypes) -> LabelInfo:
 
         raise TypeError(label_info)
 
+    def forward_tiles(self, inputs: OTXTileBatchDataEntity[SegBatchDataEntity]) -> SegBatchPredEntity:
```

Comment:
Could you add unit tests for this?

Reply:
it's covered already. I don't know why codecov hasn't updated. 
![image](https://github.com/user-attachments/assets/8695c503-ca22-49b3-ab58-5daee1ecaa3e)